src/components/tasks/card/index.tsx

```diff
@@ -36,6 +36,11 @@ const Card: FC<Props> = ({
 
```

Comment:
Good work @DeepanshuSarswat 
There is one fix that is needed to be done
Rest everything is working fine
When I select 24-08-2022 and save it, the date saved is 23-08-2022 means every time I save the date, the actual date saved is a day before 

![image](https://user-images.githubusercontent.com/78433013/181724038-e7eb9703-63d4-4bee-b600-17323f27aa1d.png)

Reply:
@rohan09-raj , I have resolve the problem.
Please have a look.